CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ## Unreleased
 
+* Remove the last of the miscellaneous jQuery ([PR #2556](https://github.com/alphagov/govuk_publishing_components/pull/2556))
```

Comment:
Not really _last_ is it? Because we still have jQuery in `modules.js`

Reply:
True, have amended. I think I went into this PR more optimistic than I've come out of it 😞